File: 1/twosum.java
import java.util.*;
class twosum
{
public static void main(String args[])
{
    Scanner in =new Scanner(System.in);
    int arr[]={1,2,3,4,5,6,7,8,9};
    System.out.println("Enter the number");
    int target=in.nextInt();
    for(int i=0;i<arr.length;i++)     //looping through the elements in array
    {
      int find = target - arr[i];      
      for(int j=0;j<arr.length;j++)    
    {   
        if(find==arr[j])              //finding the number in second loop after subracting the first index from the target
      {
          System.out.println(arr[i]+"  "+find);
        }
      }
    }
}
}

File: 1/twosum.cpp
#include <iostream>
int main(){
  std::cout << "enter the number\n";
    int target;
    std::cin >> target;
  int arr[10]={1,2,3,4,5,6,7,8,9,10};
  for(int i=0;i<=9;i++)
  {
    int find=target-arr[i];
    //std::cout << arr[i] << "\n";
    //printf("%d",arr[i]);
    printf("%d",target-arr[i]);
    for(int j=0;j<=9;j++)
    {
      if (find==arr[j]) {
        std::cout << find << "," << arr[i] << std::endl;
      } 
    }
}
}
